README.md

```diff
@@ -58,7 +58,7 @@ docker run -it --rm -p 3000:3000 -v $PWD/:/home -w /home friendsofgo/killgrave
 ```
 Remember to use the [-p](https://docs.docker.com/engine/reference/run/#expose-incoming-ports) flag to expose the container port where the application is listening (3000 by default).
 
-NOTE: If you want to use `killgrave` through Docker at the same time you use your own dockerised HTTP-based API, be careful with networking issues.
+NOTE: If you want to use `killgrave` through Docker at the same time you must use your own dockerised HTTP-based API, be careful with networking issues.
```

Comment:
Not sure about that `must` since it's not a mandatory statement.

It basically tries to say: if your application is dockerised and you want to use Killgrave also through Docker, be careful. But you can use Killgrave through Docker without the need of dockerising your application.

Reply:
Ahhh I understand, yes that change I made isn't clear, I'll work on that.

Reply:
OK I've made a change to hopefully make it clearer